swagger.yaml

```diff
@@ -0,0 +1,497 @@
+  '/appprofile/{id}':
+    get:
+      security:
+      - bearerAuth: []
+      tags:
+        - apps
+      summary: Find profile by id
+      description: ''
+      operationId: getAppProfile
+      parameters:
+        - name: id
+          in: path
+          description: id to find by
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ProfileData'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProfileData'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+  '/search/{src_text}':
+    get:
+      security:
+      - bearerAuth: []
+      tags:
+        - apps
+      summary: Find app by name
+      description: ''
+      operationId: searchBy
+      parameters:
+        - name: src_text
+          in: path
+          description: name to find by
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/AppData'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AppData'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+externalDocs:
+  description: Find out more about Swagger
+  url: 'http://swagger.io'
+servers:
+  - url: 'http://localhost:8080/api'
+  - url: 'https://localhost:8080/api'
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+  responses:
+    UnauthorizedError:
+      description: Access token is missing or invalid
+  schemas:
+    Category:
+      type: object
+      properties:
+        Id:
+          type: string
+          example: my post
+        CatName:
+          type: string
+          example: description
+        Ordinal:
+          type: string
+          example: location
+      xml:
+        name: Post
+
+      type: object
+      properties:
+        id:
+          type: string
+        src:
+          type: string
+        url:
+          type: string
+      xml:
+        name: Image
+    LoginData:
+      type: object
+      required:
+        - username
+        - password
+      properties:
+        username:
+          type: string
+        password:
+          type: string
+        issentinel:
+          type: string
+      xml:
+        name: LoginData
+    HashData:
+      type: object
+      required:
+        - password
+        - secret
+      properties:
+        password:
+          type: string
+        secret:
+          type: string
+      xml:
+        name: HashData
+    PhotoData:
+      type: object
+      required:
+      properties:
```

Comment:
missed required and properties? I get an error in swagger editor.

Reply:
You are right with all the mentioned comments, i will fix it asap. thx